Configuration/Applications/python/ConfigBuilder.py

```diff
@@ -920,6 +920,7 @@ def define_Configs(self):
         self.L1RecoDefaultCFF="Configuration/StandardSequences/L1Reco_cff"
         self.L1TrackTriggerDefaultCFF="Configuration/StandardSequences/L1TrackTrigger_cff"
         self.RECODefaultCFF="Configuration/StandardSequences/Reconstruction_Data_cff"
+        self.RECOSIMDefaultCFF="Configuration/StandardSequences/Reconstruction_MC_cff"
```

Comment:
hi @drkovalskyi @slava77 - lets discuss the naming convention.. elsewhere in StandardSequences <blah>_MC means everything in "blah" plus more stuff that is specific to MC. Thats not the case here (which I agree with that choice) so a different name is more appropriate. Eg "RecoSimMatching_cff"? (thats not perfect either)

Reply:
Since it can be more than matching I renamed it to RecoSim_cff. Hope it's Ok.